etc/ci/performance/firefox/revision_46.0.1.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
The `package.json` for the Firefox addon says version `>=38.0a1` but this filename is `revision_46.0.1.json`. Which version is needed? We should also mention the version to use in the README (instead of just `Nightly`).

Also, let's pare down the fields to just what's needed, the same as what `git_log_to_json.sh` outputs.